### ISSUE TITLE
Nested Structures don't seem to be working.

### ISSUE THREAD

**Issue by jarleif** [JiriTrecak/Laurine#18]
Nested Structures don't seem to be working in the latest version on github for Swift.
I tried both of these:
"PROFILE-NAVIGATION_BAR-ITEMS-DONE" = "Done" // -c -d -
"PROFILE.NAVIGATION_BAR.ITEMS.DONE" = "Done" // -c -d .

I tried specifying and leaving the defaults, neither worked. I even tried adding a new string to the example and that didn't work.

It changed it to /// Base translation: Thanks everyone! You rock!
    public static var ContributorsFooter : String = "Contributors.Footer".localized

**Comment by JiriTrecak** [comment]
Hello Jared,

I am sorry, seems like latest pull request broke some small piece critical functionality and I missed it when reviewing it. I pushed fix to master and tested, it now seems to work again.

Thanks for the find! Tests are coming to prevent that in the future.

Have a nice day

Jiri

**Comment by jarleif** [comment]
Hi Jiri,
Thanks for the quick fix. It seems to be working now. This makes dealing with the localized strings much easier.

Have you used the Aggregate target before? It is under the Other section when adding a new target. I setup my project so that I have an Aggregate target called Pre-Build-Scripts that I added the Laurine run script to. I then added the Pre-Build-Scripts target to be a Target Dependency on my main project. Now anytime I build my main project it will build this target first and run its run script to create my localization file. So far it seems to be working well. Doing it this way allows me to create the file before the main build is run and I can build only my strings file right from Xcode without building my whole project. Have you tried this way before? If so do you know of any issues with it?

**Comment by JiriTrecak** [comment]
Hey, good to hear. also, yes, I've used it on few projects as well - so far, there is no problem, at least from my experience. 

But if you check the example project now, I actually moved the laurine run phase before build phase in the main target and it is built first. Also, if you have whole-module-optimization enabled, it does not rebuild whole project, only the changes, so cmd+B or cmd+U is usually almost instant. Lastly, the build script in example checks if there are any changes, so laurine is only called when you make changes in localizable.strings.

Hope it helps!

Have a nice day

Jiri
